testgrid/config/config.yaml

```diff
@@ -3084,6 +3087,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-prow-canary
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+    dashboard_tab_alert_options:
+      alert_mail_to_addresses: michelle.192837+alerts@gmail.com
```

Comment:
you are not likely to receive emails for this job... I'll find you a better(flakier) one

Reply:
(try http://k8s-testgrid.appspot.com/master-upgrade#gce-1.7-master-upgrade-master), and add lusensjtu+alerts@gmail.com as well :-)

Reply:
Done?